react/features/welcome/components/styles.js

```diff
@@ -171,6 +171,13 @@ export default createStyleSheet({
         flexDirection: 'column'
     },
 
+    /**
+     * Container of the side bar.
+     */
+    sideBar: {
+        width: 250
```

Comment:
use `SIDEBAR_WIDTH` ?

Reply:
I used to use that before it was refactored, because it was used in many cases. Now the slider component became size agnostic, so it makes no sense to me to put it in a const and use it only here. Although I forgot to remove the original constant from the code, so I did that now, thanks.